include/varcache.h

```diff
@@ -1,20 +1,24 @@
 #include <usual/strpool.h>
 
+#define MAX_NUM_CACHE_VARS 100
```

Comment:
Let's remove this and make the size dependent on the variables that are configured in the ini file.